src/librustc_mir/interpret/step.rs

```diff
@@ -74,7 +74,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         Ok(true)
     }
 
-    fn statement(&mut self, stmt: &mir::Statement<'tcx>) -> InterpResult<'tcx> {
+    crate fn statement(&mut self, stmt: &mir::Statement<'tcx>) -> InterpResult<'tcx> {
```

Comment:
Hm... of it's (semi-)public, maybe it should have a doc comment?

Also for consistency, `terminator` should probably get the same treatment.